cmd/harmony/main.go

```diff
@@ -309,10 +309,9 @@ func setupNodeAndRun(hc harmonyConfig) {
 				utils.Logger().Warn().Str("signal", sig.String()).Msg("Gracefully shutting down...")
 				const msg = "Got %s signal. Gracefully shutting down...\n"
 				fmt.Fprintf(os.Stderr, msg, sig)
-				// stop block proposal service for leader
-				if node.Consensus.IsLeader() {
-					node.ServiceManager().StopService(service.BlockProposal)
-				}
```

Comment:
why removed this part?

Reply:
Since `serviceManager.Stop()` in `node.ShutDown()` will stop all services registered, including BlockProposal.